spec/ParseFile.spec.js

```diff
@@ -860,4 +860,162 @@ describe('Parse.File testing', () => {
       });
     });
   });
+
+  describe('disable file upload', () => {
+    it('can reject file upload with unspecified', async () => {
+      await reconfigureServer({
+        fileUpload: {},
+      });
+      try {
```

Comment:
may add a test just for DX about behavior when fileUpload is set to `undefined`

Reply:
Rather than setting it to `undefined` I suggest to add a test case where `fileUpload` is not set at all. Setting it to `undefined` makes it declared, which is not he same.